x/epochs/keeper/epoch_infos.go

```diff
@@ -9,9 +12,28 @@ import (
 
 // GetEpochInfo returns epoch info by identifier
 func (k Keeper) GetEpochInfo(ctx sdk.Context, identifier string) (types.EpochInfo, bool) {
```

Comment:
I kept the human-readable identifier, so it is easy for devs to use the epochs - I assume this is the reason human-readable identifiers were used instead of the actual duration.
E.g. easier to hook the incentives on `week` than to compute duration.
https://github.com/tharsis/evmos/blob/a4e95a11ac00cf492087c94e05abf8a5a1029e67/x/incentives/types/params.go#L47

So I introduced two KV stores (negligible additional state):
- identifier => duration
- duration => EpochInfo
And I made sure epochs are sorted numerically, after duration.

Reply:
@fedekunze I will remove the human-readable identifiers.
Duration-based identifiers can be imported from the epochs module as enums.

Reply:
@fedekunze, note that if I remove the human-readable identifiers, `evmosd q epochs current-epoch week` will not work anymore. You will need to do something like `evmosd q epochs current-epoch "168h0m0s"`.

Reply:
could we use the map identifier -> duration?

Reply:
Yes, this is solved already: `IdentifierToDuration` map exists. So other modules can still use identifiers like `week`.